invesalius/gui/dialogs.py

```diff
@@ -3051,22 +3049,22 @@ def _init_gui(self):
         main_sizer.Fit(self)
         main_sizer.SetSizeHints(self)
 
-    def StartTimer(self):
+    def StartTimer(self) -> None:
         self.t0 = time.time()
 
-    def StopTimer(self):
+    def StopTimer(self) -> None:
         fmt = "%H:%M:%S"
         self.lbl_time.SetLabel(time.strftime(fmt, time.gmtime(time.time() - self.t0)))
         self.progress.SetValue(0)
 
-    def Pulse(self):
+    def Pulse(self) -> None:
         fmt = "%H:%M:%S"
         self.lbl_time.SetLabel(time.strftime(fmt, time.gmtime(time.time() - self.t0)))
         self.progress.Pulse()
 
 
 class FFillOptionsDialog(wx.Dialog):
-    def __init__(self, title, config):
+    def __init__(self, title: str, config: "FFillSegmentationConfig"):
```

Comment:
config: str

Reply:
I think I made a mistake in this one and it is supposed to be `FFillConfig` not `FFillSegmentationConfig`.

Reply:
Sorry, my mistake. It's a `FFillConfig`